packages/terra-form-input/src/Input.module.scss

```diff
@@ -87,15 +87,15 @@
 
     &:focus {
       background-color: var(--terra-form-input-error-focus-background-color, #fff);
-      background-size: var(--terra-form-input-error-focus-keyboard-animation-size);
+      background-size: var(--terra-form-input-error-focus-keyboard-animation-background-size);
       border: var(--terra-form-input-error-focus-border, 1px solid #4cb2e9);
 
       [dir=ltr] & {
-        box-shadow: var(--terra-form-input-error-focus-keyboard-box-shadow-ltr);
+        box-shadow: var(--terra-form-input-error-focus-keyboard-ltr-box-shadow);
```

Comment:
nit: I'd drop `keyboard` from this variable name and the one below it

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/d89bf406444dc93fa1237dae7fc382f0ec92b6b0#diff-462afbdb742ea35bac7504edbfd87d7dR94